lib/linux/background-managers/xfconf-query.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const {commandExists, execFile} = require('../util');
+
+exports.isAvailable = function () {
+	return commandExists('xfconf-query');
+};
+
+exports.set = async function (imagePath) {
+	await execFile('xfconf-query', ['-c xfce4-desktop', '-p /backdrop/screen0/monitor0/image-path', '-s ' + imagePath]);
```

Comment:
Use long flags instead of short flags

Use template literal

And the it should probably be `'-c xfce4-desktop',` => `'-c=xfce4-desktop',`?